tests/resources/generate-outputs.sh

```diff
@@ -7,9 +7,24 @@ ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
 cd "$ROOT_DIR"
 
-mtt train options.yaml -o model-32-bit.pt -r base_precision=32
-mtt train options.yaml -o model-64-bit.pt -r base_precision=64
-mtt train options-pet.yaml -o model-pet.pt
+FORCE_REGENERATE=false
```

Comment:
We also want to regenerate if the code changed, in case one of the change broke the tests

Reply:
makes sense, but this is a bit trickier? rn I pushed something relative to the last commit but then of course it doesn't run if the user ended up making an uncommitted change which would have broken tests..